notebooks/Setting up a labeling task.ipynb

```diff
@@ -128,18 +128,14 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        # need to double check if these changes work!!!!!
</pre><p>Was this checked?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>eventually </p>